simplematrixbotlib/match.py

```diff
@@ -84,15 +86,16 @@ def __init__(self, room, event, bot, prefix="") -> None:
         self._prefix = prefix
 
         """Forms of identification"""
-        self._own_user_id = f"@{self._bot.creds.username}:{self._bot.creds.homeserver.replace("https://","").replace("http://","")}"
-        self._own_nio_user = self.room.users[own_user_id]
-        self._own_disambiguated_name = own_nio_user.disambiguated_name
-        self._own_display_name = own_nio_user.display_name
-        self._own_pill = f"<a href=\"https://matrix.to/#/{self.room.own_user_id}\">"
+        self._own_user_id = room.own_user_id
+        self._own_nio_user = self.room.users[self._own_user_id]
+        self._own_disambiguated_name = self._own_nio_user.disambiguated_name
+        self._own_display_name = self._own_nio_user.display_name
+        self._own_display_name_colon = f"{self._own_display_name}:"
```

Comment:
why do you allow colon only for _own_display_name? it is reasonable to expect always for mentioning, e.g. element inserts it after a pill

Reply:
I didn't realize that it also applied to the rest.